code/datums/helper_datums/getrev.dm

```diff
@@ -89,6 +89,7 @@
 	msg += "Protect Authority Roles From Traitor: [CONFIG_GET(flag/protect_roles_from_antagonist)]"
 	msg += "Protect Assistant Role From Traitor: [CONFIG_GET(flag/protect_assistant_from_antagonist)]"
 	msg += "Protect Command Roles From Traitor: [CONFIG_GET(flag/protect_heads_from_antagonist)]"
+	msg += "Protect Exploration From Traitor: [CONFIG_GET(flag/protect_exploration_from_antagonist)]"
```

Comment:
Do I understand correctly that this is the only one of these that protects from antagonist *targets* too? If yes, maybe it should be noted in the message?

Reply:
Its in the config option as that.

Reply:
updated anyway cuz why not